lib/exercises.rb

```diff
@@ -1,20 +1,63 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: 0(1)
+# Space Complexity: 0(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 , however your time complexity is off.  You loop through all the strings so it's a minimum of O(n).  If you assume the strings are all small then the sorting can be ignored.